spec/halogen/links_spec.rb

```diff
@@ -68,6 +68,36 @@
   end
 
   describe Halogen::Links::InstanceMethods do
+    describe "options[:include_links]" do
+      let :klass do
+        Class.new do
+          include Halogen
+        end
+      end
+      
+      it "includes links when true" do
+        link = klass.link(:self, :templated, foo: 'foo', attrs: { bar: 'bar' }) { 'path' }
+        repr = klass.new('include_links'  => true)        
```

Comment:
we want to add one without anything at all  for defaults :)

Reply:
good call!